src/hooks/api/useInvestments.ts

```diff
@@ -36,20 +38,28 @@ export function useInvestments<TData = InvestmentAccount[]>(
 ): UseQueryResult<TData> {
   const { data: interval } = useInterval();
 
-  const { data: accounts } = useAccounts();
+  const { data: accounts, dataUpdatedAt: accountsUpdatedAt } = useAccounts();
+  const { data: splits, dataUpdatedAt: splitsUpdatedAt } = useSplits({ type: 'INVESTMENT' });
```

Comment:
bringing splits back into the query so values are refreshed when splits are updated